code/game/objects/items/twohanded.dm

```diff
@@ -346,7 +346,8 @@
 		icon_state = "dualsaber[item_color][wielded]"
 	else
 		icon_state = "dualsaber0"
-	SEND_SIGNAL(src, COMSIG_COMPONENT_CLEAN_ACT, CLEAN_STRENGTH_BLOOD)
```

Comment:
Comsig

Reply:
This doesn't need to be handled. It's purely there only as an overlay cleanup.